api/app/signals/apps/signals/models/area.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import F, Max, Min
+from django.contrib.gis.db import models
+
+# TODO: slugfields for the "code" fields below
+
+
+class AreaManager(models.Manager):
+    # Only show the most recent properties for an area
+    def get_queryset(self):
```

Comment:
Overwriting the whole query set? Is this necessary? It feels like this will hold us back in selecting what we really need. Not sure about the implementation here.

Reply:
This was an experiment, to see whether we could avoid duplicating data (the timestamps) between the various models. It works for the model manager methods, see `Area.objects.get` in the tests, but not for instances created by hand with `Area(...)`.